Android/app/src/androidTest/java/PageObjects/TestApplication.kt

```diff
@@ -0,0 +1,31 @@
+package PageObjects
+
+import android.support.test.InstrumentationRegistry
+import android.content.Intent
+
+/**
+ * Created by bpage on 2/21/18.
+ */
+
+class TestApplication {
```

Comment:
This can be subclassed as `IdentityProviderApplication` and `ServiceProviderApplication` for IDP tests in the future.